packages/default-theme/components/SwTopNavigation.vue

```diff
@@ -13,21 +13,17 @@
       </template>
       <template #navigation>
         <SfHeaderNavigationItem
-          v-for="{ routeLabel, routePath } in routes"
-          :key="routeLabel"
+          v-for="category in navigationElements"
+          :key="category.name"
           class="sf-header__link"
+          @mouseover="setHoverNavigationItem(category.name)"
+          @mouseleave="setHoverNavigationItem('')"
+          @keyup.tab="setHoverNavigationItem(category.name)"
         >
-          <nuxt-link class="sf-header__link" :to="routePath">
-            <a
-              :style="{
-                display: 'flex',
-                alignItems: 'center',
-                height: '100%'
-              }"
-            >
-              {{ routeLabel }}
-            </a>
+          <nuxt-link class="sf-header__link" :to="`/${category.route.path}`">
```

Comment:
It would be better to have the `path` and any other deep properties using helpers package. That's the layer to transform or extract SW specific nested data. it could be named for instance `getNavigationElements`, and on the occasion, you could create the interface for global (next projects) purposes ;) so, two birds killed with the one stone :smiley: 
less logic in theme is always better :blue_heart:

Reply:
You are completely right